PowerEditor/src/Notepad_plus_Window.h

```diff
@@ -57,6 +57,11 @@ filePath : file or folder name to open (absolute or relative path name)\r\
 class Notepad_plus_Window : public Window
 {
 public:
+	static Notepad_plus_Window & getInstance() {
+		static Notepad_plus_Window * instance = new Notepad_plus_Window;

```

Comment:
`free` is missing.

Reply:
That was the same Singleton technique as the one used in `NppParameters`. Still to be identical in behaviour, I switched to `std::unique_ptr` in `wWinMain()`.